container/id/id.go

```diff
@@ -113,3 +113,11 @@ func (id *ID) DecodeString(s string) error {
 func (id ID) String() string {
 	return id.EncodeToString()
 }
+
+// FromBinary calculates identifier of the binary-encoded container
+// in CAS of the NeoFS containers and writes it into dst.
```

Comment:
what `dst` is here?

Reply:
I think, `id` will be right destination. Updated